protocol/0009-MRKP-mark_price.md

```diff
@@ -6,70 +6,81 @@ A *Mark Price* is a concept derived from traditional markets.  It is a calculate
 
-## Usage of the *Mark Price*
 
-The most recently calculated *Mark Price* is used in the [mark-to-market settlement](./0003-MTMK-mark_to_market_settlement.md).  A change in the *Mark Price* is one of the triggers for the mark-to-market settlement to run.
+### 2. Flexible mark price methodology
 
-## Algorithms for calculating the *Mark Price*
+The calculations are specified in [markprice methodology research note](https://github.com/vegaprotocol/research/blob/markprice-updates/papers/markprice-methodology/markprice-methodology.tex).
+Here, we only write the acceptance criteria.
+Note that for calculating the median with an even number of entries we sort, pick out the two values that are in the middle of the list and average those. So in particular with two values a median is the same as the average for our purposes.
 
-### 1. Last Traded Price of a Sequence with same time stamp with maximum frequency set by `network.markPriceUpdateMaximumFrequency`
 
- The mark price for the instrument is set to the last trade price in the market following processing of each transaction (i.e. submit/amend/delete order) from a sequence of transactions with the same time stamp, provided that at least `network.markPriceUpdateMaximumFrequency` has elapsed since the last mark price update.
+## Acceptance criteria
 
- >*Example:* Assume `network.markPriceUpdateMaximumFrequency = 10s`.
+- The mark price must be set when the market leaves opening auction. (<a name="0009-MRKP-002" href="#0009-MRKP-002">0009-MRKP-002</a>)
+- Each time the mark price changes the market data event containing the new mark price should be emitted.Specifically, the mark price set after leaving each auction, every interim mark price as well as the mark price based on last trade used at market termination and the one based on oracle data used for final settlement should all be observable from market data events. (<a name="0009-MRKP-009" href="#0009-MRKP-009">0009-MRKP-009</a>)
 
- Consider the situation where the mark price was last updated to $900 and this was 12s ago. There is a market sell order for -20 and a market buy order for +100 with the same time stamp. The sell order results in two trades: 15 @ 920 and 5 @ 910. The buy order results in 3 trades; 50 @ $1000, 25 @ $1100 and 25 @ $1200, the mark price changes **once** to a new value of $1200.
+### Algorithm 1 (last trade price, excluding network trades):
 
- Now 8s has elapsed since the last update and there is a market sell order for volume 3 which executes against book volume as 1 @ 1190 and 2 @ 1100.
- The mark price isn't updated because `network.markPriceUpdateMaximumFrequency = 10s` has not elapsed yet.
+- If `network.markPriceUpdateMaximumFrequency=0s` then any transaction that results in one or more trades causes the mark price to change to the value of the last trade and only the last trade. (<a name="0009-MRKP-003" href="#0009-MRKP-003">0009-MRKP-003</a>)
```

Comment:
I think this is no longer allowed (we don't use it and it would cause division by 0 in the new methodology). Cc @ze97286 . 

I think this AC should be removed.